html/changelogs/burgerbb-junkfood.yml

```diff
@@ -0,0 +1,11 @@
+author: BurgerBB
+delete-after: True
+changes: 
+  - rscadd: "Added several new chemicals; Palm Oil, Shortening, High Fructose Corn Syrup, and MSG. These chemicals can be found in junkfood."
```

Comment:
Short that changelog.
Details can be found on github for each PR.
Also, dont put in the full link. The PR Number is sufficient.